### PR TITLE
jsDelivr related changes

### PR DESCRIPTION
**Pull request by jimaek**
- Migrated privacy policy to jsDelivr
- Removed jobs
- Switched GA
- Removed some affiliate stuff

### CODE REVIEW

**Comment by jimaek** [comment]
@XhmikosR Sorry I just noticed the related changes over here https://github.com/MaxCDN/bootstrapcdn/pull/1675
Let me know what you want to do. I could remove my test changes from here

File: app.js
Comment (XhmikosR) [inline app.js:135]: I'd add redirects for the removed pages

File: views/_partials/footer.pug
Comment (XhmikosR) [inline views/_partials/footer.pug:25]: Please add `rel='noopener'` and remove the stray whitespace

File: views/_partials/jumbotron.pug
Comment (XhmikosR) [inline views/_partials/jumbotron.pug:13]: Same here

**Comment by XhmikosR** [comment]
@jimaek yeah, I'd prefer if the test changes were made in #1675.

BTW the content types should match what we had. Personally, I prefer being spec-compliant and we've gone to great lengths all these year to do this.

**Comment by jimaek** [comment]
I made the changes, but I guess the tests PR needs to get merged first to get the real test results here.

**Comment by XhmikosR** [comment]
The CDN tests yeah. I think better finish with #1675 first.

File: views/_partials/jumbotron.pug
Comment (XhmikosR) [inline views/_partials/jumbotron.pug:5]: Not sure if you want to link to jsdevilr here too or maybe https://en.wikipedia.org/wiki/Content_delivery_network? It's nitpicking, so it's your call :)

Comment (jimaek) [reply views/_partials/jumbotron.pug:5]: Yeah maybe in the future. For now let's just do a cleanup :)

File: app.js
Comment (XhmikosR) [inline app.js:137]: This redirect is missing @jimaek

**Comment by XhmikosR** [comment]
The Heroku preview does not fire, and I can't seem to be able to manually trigger it...

File: config/helmet-csp.js
Comment (XhmikosR) [inline config/helmet-csp.js:40]: We should check if the other entries are all needed, but I don't have the time to do it now.

Comment (jimaek) [reply config/helmet-csp.js:40]: Lets do things in small and quick steps. As we remove more pages we can review this and update it too.
I think this PR is ready and after that develop can be merged into production.

Comment (XhmikosR) [reply config/helmet-csp.js:40]: Sure, just wanted to note the issue :P

**Comment by XhmikosR** [comment]
The CDN has some issues that probably stem from the origin... I'll restart the run, but it seems that it's super slow when hitting the origin or something like that.

>         https://stackpath.bootstrapcdn.com/bootstrap/4.5.1/css/bootstrap-reboot.css
>           it works:
>     Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/home/runner/work/bootstrapcdn/bootstrapcdn/test/functional_test.js)

**Review by XhmikosR — APPROVED** [review]

**Comment by jimaek** [comment]
> The CDN has some issues that probably stem from the origin... I'll restart the run, but it seems that it's super slow when hitting the origin or something like that.
> 
> > ```
> >     https://stackpath.bootstrapcdn.com/bootstrap/4.5.1/css/bootstrap-reboot.css
> >       it works:
> > Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/home/runner/work/bootstrapcdn/bootstrapcdn/test/functional_test.js)
> > ```

Yeah uncached will be slow unfortunately(but not 10s). We utilize an additional layer between CF and S3 because CF cant handle S3 easily by itself. Thats why I wanted to review the slowness separately after we finish changing stuff and purging the cache.

**Pull request merged**
